content-repo/extra-docs/releases/20.7.0.md

```diff
@@ -0,0 +1,516 @@
+Added the *Tags* parameter.
+
+---
+
+### ArcSight ESM Pack v1.0.3
+#### Integrations
+##### ArcSight ESM v2  
+Fixed a bug in which some fetches returned duplicate alerts.
+Added a new integration parameter *Use REST Endpoints* which enables usage of REST endpoints for the ***as-get-entries*** and ***as-clear-entries*** commands.
+
+---
+
+### Attivo Botsink Pack v1.0.1
+#### Integrations
+##### Attivo Botsink  
+Fixed an issue where errors were not handled as expected.
+
+---
+
+### Bambenek Consulting Feed Pack v1.0.1
+#### Integrations
+##### Bambenek Consulting Feed  
+Added the *Tags* parameter.
+
+---
+
+### Base Pack v1.1.0
+#### CommonServerPowerShell
+#### Scripts
+##### SaneDocReports  
+- Fixed SVG image rendering in doc reports.
+- Added the ability to add customer logos to doc reports.
+- Reverted changes made in 1.0.12
+
+##### SanePdfReports  
+- Fixes word overlapping in graphs.
+- Rolled back the docker image to fix a conflict issue.
+- Updated the sane-pdf-reports docker tag which fixes the graph labels overlap bug.
+
+##### CommonServerPython  
+- Fixed an issue where the ***to_context*** function did not return the proper outputs when the **CommandResult** object was supplied with only ***readable_outputs***.
+ - Fixed and issue where ***to_context*** function returned null instead of an empty list when supplied with empty outputs.
+Added wrapper functions for getting and setting integration context.
+
+---
+
+### CSV Feed Pack v1.0.2
+#### Integrations
+##### CSVFeed  
+Added the *Tags* parameter.
+
+---
+
+### Carbon Black Enterprise Protection Pack v1.0.2
+#### Playbooks
+##### Carbon black Protection Rapid IOC Hunting  
+Updated the playbook to use the **Carbon Black Enterprise Protection v2** integration .
+
+---
+
+### Check Point Firewall Pack v1.0.3
+#### Integrations
+##### Check Point  
+Fixed an issue where the ***checkpoint*** command did not work as expected.
+
+---
+
+### Common Playbooks Pack v1.5.1
+#### Playbooks
+##### Block IP - Generic v2  
+Added FortiGate Ban IP command to Block IP - Generic v2 playbook.
+
+---
+
+### Common Scripts Pack v1.2.3
+#### Scripts
+##### ParseEmailFiles  
+- Fixed an issue where errors were not handled as expected.
+- Fixed an issue where .eml files with the content type "message/rfc822" were not recognized as expected.
+
+##### SetGridField  
+Fixed an issue where the script failed on `mixed-types` error.
+
+##### JSONtoCSV  
+Improved error message when invalid JSON entry is given.
+
+---
+
+### Common Types Pack v1.4.0
+#### IncidentFields
+#### IndicatorFields
+#### Layouts
+##### domainRepUnified  
+Added the Feed Related Indicators section to the layout.
+
+##### unifiedFileRep  
+Added the Feed Related Indicators section to the layout.
+
+##### ipRep  
+Added the Feed Related Indicators section to the layout.
+
+##### urlRep  
+Added the Feed Related Indicators section to the layout.
+
+---
+
+### EWS Pack v1.1.3
+#### Integrations
+##### EWSO365  
```

Comment:
```suggestion
##### New: EWSO365  
```
@yaakovi fyi

Reply:
is this new? didnt we release it last time?

Reply:
yes, it is. removing